airbyte-server/src/main/java/io/airbyte/server/apis/ConfigurationApi.java

```diff
@@ -727,6 +728,11 @@ public ConnectionState getState(final ConnectionIdRequestBody connectionIdReques
     return execute(() -> schedulerHandler.getState(connectionIdRequestBody));
   }
 
+  @Override
+  public ConnectionStateType getStateType(final ConnectionIdRequestBody connectionIdRequestBody) {
+    return null;
+  }
```

Comment:
Just checking - do we have an issue for implementing this endpoint specifically? Looking at the [whimsical board](https://whimsical.com/board-for-notes-Tt8Z7ZADi5ECh4FvatQbsp), I only see issues for implementing the CatalogDiff behavior

Reply:
I was bundling it in with this issue https://github.com/airbytehq/airbyte/issues/13648.